diff --git a/rmw_implementation/CMakeLists.txt b/rmw_implementation/CMakeLists.txt
index c23861e..784ddb5 100644
--- a/rmw_implementation/CMakeLists.txt
+++ b/rmw_implementation/CMakeLists.txt
@@ -40,7 +40,7 @@ message(STATUS "")
 
 # if only a single rmw impl. is available or poco is not available
 # this package will directly reference the default rmw impl.
-if(NOT RMW_IMPLEMENTATIONS MATCHES ";" OR NOT Poco_FOUND)
+if(FALSE)  #NOT RMW_IMPLEMENTATIONS MATCHES ";" OR NOT Poco_FOUND)
   set(RMW_IMPLEMENTATION_SUPPORTS_POCO FALSE)
 
 else()
diff --git a/rmw_implementation/src/functions.cpp b/rmw_implementation/src/functions.cpp
index 45c4137..3e93d35 100644
--- a/rmw_implementation/src/functions.cpp
+++ b/rmw_implementation/src/functions.cpp
@@ -18,6 +18,7 @@
 
 #include <list>
 #include <string>
+#include <iostream>
 
 #include <fstream>
 #include <sstream>
@@ -123,7 +124,9 @@ get_library()
     if (env_var.empty()) {
       env_var = STRINGIFY(DEFAULT_RMW_IMPLEMENTATION);
     }
-    std::string library_path = find_library_path(env_var);
+    // std::string library_path = find_library_path(env_var);
+    std::string library_path = "/opt/ros/crystal/lib/librmw_fastrtps_cpp.so";
+    std::cout << "shared lib: " << library_path << "\n";
     if (library_path.empty()) {
       RMW_SET_ERROR_MSG(
         ("failed to find shared library of rmw implementation. Searched " + env_var).c_str());
